src/mouse-touchpad.vala

```diff
@@ -99,6 +99,7 @@ namespace MouseTouchpad {
             search_results.set ("%s → %s".printf (display_name, _("Primary button")), "");
             search_results.set ("%s → %s".printf (display_name, _("Reveal pointer")), "");
             search_results.set ("%s → %s".printf (display_name, _("Middle click paste")), "");
+            search_results.set ("%s → %s".printf (display_name, _("Hold to secondary click")), "");
```

Comment:
This should probably match the switch label right?

Reply:
Ah yeah good catch. I played with a few different labels.

Reply:
```suggestion
            search_results.set ("%s → %s".printf (display_name, _("Long-press secondary click")), "");
```